scripts/V470__AJUSTE_REMOVER_PENDENCIAS_FECHAMENTO_INDEVIDAS_EJA.sql

```diff
@@ -0,0 +1,23 @@
+delete 
+  from pendencia_fechamento where id in (select distinct (pf.id)
+  										from pendencia_fechamento pf  
+									   inner join pendencia p on p.id = pf.pendencia_id 
+									   inner join fechamento_turma_disciplina ftd on ftd.id = pf.fechamento_turma_disciplina_id 
+								       inner join fechamento_turma ft on ft.id = ftd.fechamento_turma_id 
+									   inner join turma t on t.id = ft.turma_id 
+									   where p.tipo = 4
+									     and t.modalidade_codigo = 3
+									     and ftd.disciplina_id in (1060,1061));
```

Comment:
sera que não vale a pena trocar o ftd.disciplina_id in (1060,1061) por um join com componente_curricular e filtrar permite_registro_frequencia ?

Reply:
Nesse caso não, pq os componentes permitem registro de frequência, porém pra EJA é uma exceção, mesmo que permita, não pode registrar nem frequência nem pendências de fechamento